pkg/common/cns-lib/volume/listview_if.go

```diff
@@ -0,0 +1,24 @@
+package volume
+
+import (
+	"context"
+
+	"github.com/vmware/govmomi/vim25/types"
+
+	cnsvsphere "sigs.k8s.io/vsphere-csi-driver/v2/pkg/common/cns-lib/vsphere"
+)
+
+// ListViewIf provides methods to start and modify a listView object to monitor CNS tasks.
+// NewListViewImpl satisfies ListViewIf and should be initialized to use the interface methods
+type ListViewIf interface {
```

Comment:
I am assuming the `If` postfix is to indicate it as an interface? here are some recommendation on naming interfaces: https://medium.com/@dotronglong/interface-naming-convention-in-golang-f53d9f471593

Reply:
I read through the blog. The `If` suffix ends up being a shorter version of the `Interface` suffix option which also seems to satisfy the 'shorter is better' golang rule mentioned by the author.